modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -13,21 +13,45 @@ You can also grant developers and other users different permissions:
 * To monitor user-defined projects.
 * To configure the components that monitor user-defined projects.
 * To configure alert routing for user-defined projects.
+* To manage alerts and silences for user-defined projects.
 
-You can grant the permissions by assigning one of the following monitoring roles:
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
```

Comment:
Should this be _or_, as the user selects only one?
 ```suggestion
You can grant the permissions by assigning one of the following monitoring roles or cluster roles:
```

Reply:
Hello @mburke5678 I changed it to "or" as you suggested. Is everything else okay (I see that `peer-review-done` label is not applied yet. Thanks!